lua/lint/linters/solhint.lua

```diff
@@ -0,0 +1,61 @@
+local vim = vim
+
+local severities = {}
+severities[2] = vim.diagnostic.severity.ERROR
+severities[3] = vim.diagnostic.severity.WARN
+
+return {
+  cmd = "solhint",
+  stdin = false,
+  args = {
+    "-f",
+    "json"
+  },
+  ignore_exitcode = true,
+  parser = function(output, bufnr)
+    local diagnostics = {}
+    if output == "" then
+      print("solhint: parse error")
+      return diagnostics
+    end
+
+    if output == "Failed to load a solhint's config file." then
+      print(output)
+      return diagnostics
+    end
+
+    local buffer_path = vim.fn.fnamemodify(vim.api.nvim_buf_get_name(bufnr), ":~:.")
```

Comment:
I think the `fnamemodify` call might cause problems if users set a custom `cwd` when calling `try_lint` or `lint`.